### DISCUSSION TITLE
Configuracion DSLR Canon 450D

### DISCUSSION THREAD

**Discussion by jdemula** [aaronwmorris/indi-allsky#564] [Q&A] [answered]
Hello everyone.
First of all, thank you for this magnificent work and that you are sharing.
Also apologies in advance if something is not understood (Google translation).
I am an amateur astronomer, collaborator of the AAVSO (DJEI).
I have been using the Astroberry distribution for a long time and with another Raspberry PI4 and an old T7 camera (Daytson) I installed Thomas Jacquin's Allsky software but I missed the functions that I am seeing here, and above all, I wanted to give my old canon camera a better use 450D and a Sigma fisheye lens that I have from my days as a virtual tour photographer.
concretizing I have not just made it work although the system recognizes the camera and model. I have put the camera's own parameters in the specific configuration:
{
     "PROPERTIES": {
         "CCD_INFO": {
             "CCD_MAX_X": 4272,
             "CCD_MAX_Y": 2848,
             "CCD_PIXEL_SIZE": 5.19,
             "CCD_BITSPERPIXEL": 14
         }
     },
     "SWITCH": {}
But it returns the following error in the log:
**"[INFO] MainProcess indi.newMessage() #325: new Message 2023-02-13T23:20:15: Error: Invalid range for Pixel size X (CCD_PIXEL_SIZE_X). Valid range is from 1 to 40. Requested value is 0"**
And even if I modify the "CCD_PIXEL_SIZE" parameter even without decimals, I still get the same error and the process breaks down.
If you want I can pass the complete log.
Greetings

**Comment by aaronwmorris** [comment, marked as answer]
It has been a while since I verified using my Canon camera, but we can figure it out together.

Try adding the following properties.  Ultimately, I do not think the actual values have to be absolutely correct, they just have to be defined.

```json
{
    "PROPERTIES" : {
        "CCD_INFO" : {
            "CCD_MAX_X": 4272,
            "CCD_MAX_Y": 2848,
            "CCD_PIXEL_SIZE" : 4.3,
            "CCD_PIXEL_SIZE_X" : 4.3,
            "CCD_PIXEL_SIZE_Y" : 4.3,
            "CCD_BITSPERPIXEL" : 14
        }
    },
    "SWITCHES" : {}
}
```

**Reply by jdemula** [reply]
Wow! Is working. Now I just have to continue with the fine adjustments. Perfect. Thank you so much !

**Reply by aaronwmorris** [reply]
Great!  I will update my wiki with the additional options.
